src/net/udp/mod.rs

```diff
@@ -297,12 +300,11 @@ impl UdpSocket {
     /// #
     /// use async_std::net::UdpSocket;
     ///
-    /// let socket = UdpSocket::bind("127.0.0.1:0").await?;
+    /// let socket = UdpSocket::bind("127.0.0.1:7878").await?;
     /// socket.connect("127.0.0.1:8080").await?;
+    /// let bytes = socket.send(b"Hi there!").await?;
     ///
-    /// let mut buf = vec![0; 1024];
-    /// let n = socket.recv(&mut buf).await?;
```

Comment:
Why remove `recv` call?
Is this an example of the `recv` function?

Reply:
Good catch. It looks like I put the docs in the wrong spot. I'll correct that issue right now

Reply:
Fixed it